Pop-Grupa-A-Backend/app/main/services/storage_service.py

```diff
@@ -0,0 +1,16 @@
+import requests
+#todo move to env
+STORAGE_URL = '0.0.0.0:8000'
+STORAGE_APP_URL = STORAGE_URL + "/app"
+STORAGE_TASK_URL = STORAGE_URL +"/task"
+
+def getApplication(app_id):
```

Comment:
Gdzie te metody są uzywane? i czy to jest potrzebne?

Reply:
w końcu nie są, usunełam